src/server/engine_shard.cc

```diff
@@ -420,11 +422,14 @@ void EngineShard::StartPeriodicHeartbeatFiber(util::ProactorBase* pb) {
     return;
   }
   auto heartbeat = [this]() { Heartbeat(); };
+  auto priority = absl::GetFlag(FLAGS_background_heartbeat) ? fb2::FiberPriority::BACKGROUND
+                                                            : fb2::FiberPriority::NORMAL;
 
   std::chrono::milliseconds period_ms(*cycle_ms);
 
+  fb2::Fiber::Opts fb_opts{.priority = priority, .name = "heatbeat"};
   fiber_heartbeat_periodic_ =
-      MakeFiber([this, index = pb->GetPoolIndex(), period_ms, heartbeat]() mutable {
+      fb2::Fiber(fb_opts, [this, index = pb->GetPoolIndex(), period_ms, heartbeat]() mutable {
```

Comment:
providing a name info fb_opts and then overriding it below is confusing, so let's fix fiber interface.